osbenchmark/resources/custom-test-procedures.json.j2

```diff
@@ -0,0 +1,58 @@
+{ 
+  "name": "append-no-conflicts",
+  "schedule": [
+    {
+      "operation": "delete-index"
+    },{% raw %}
+    {
+      "operation": {
+        "operation-type": "create-index",
+        "settings": {{index_settings | tojson}}
+      }
+    },{% endraw %}
+    {
+      "operation": {
+        "operation-type": "cluster-health",
+        "index": {{ indices | map(attribute='name') | list | join(',') | tojson }},{% raw %}
+        "request-params": {
+          "wait_for_status": "{{cluster_health}}",
+          "wait_for_no_relocating_shards": "true"
+        },
+        "retry-until-success": true
+      }
+    },
+    {
+      "operation": "index-append",
+      {% raw -%}"clients": {{bulk_indexing_clients}},
+      "ignore-response-error-level": "{{error_level}}"{% endraw -%}
+    },
+    {
+      "name": "refresh-after-index",
+      "operation": "refresh"
+    },
+    {
+      "name": "search-after-index",
+      "operation": "search"
+    },
+    {% endraw -%}
+      {%- block queries -%}
+        {% for query in custom_queries %}
+        {
+          "operation":"{{query.name}}", 
+          "warmup-iterations": {{query.warmup_iterations}}}},
+          "iterations": {{query.iterations}},
+          {%- if query.target_throughput %}
+          "target-throughput": {{ query.target_throughput}}
+          {%- else %}
+          "target-throughput": {{ query.target_throughput | tojson }}
+          {%- endif %}
+          {%-if search_clients is defined and search_clients %}
+          ,"clients": {{ search_clients | tojson}}
+          {%- endif %}
+        },{% if not loop.last %},{% endif -%}
+        {% endfor %}
```

Comment:
This is what we should also include in the default-test-procedures operation `search-after-index`